plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-manager.ts

```diff
@@ -404,6 +435,32 @@ export class SessionManager {
         }
     }
 
+    private async startRealtime() {
+        if (this.realtime) {
+            return
+        }
+
+        status.info('⚡️', `blob_ingester_session_manager Real-time mode started `, { sessionId: this.sessionId })
```

Comment:
we should gauge here so we can say how many concurrent sessions are playing back from redis